Clean lib auth

Beagle opened the pull request:
Round 1 of lib_auth clean up.

tchalvak commented:
Branch seems to fail, fatal error on login attempt:

( ! ) Fatal error: Class 'app\Controller\Request' not found in /srv/ninjawars/deploy/lib/control/LoginController.php on line 86
Call Stack
#	Time	Memory	Function	Location
1	0.0220	3618376	{main}( )	.../login.php:0
2	0.0249	4187312	app\Controller\LoginController->requestLogin( )	.../login.php:15
3	0.0253	4188784	app\Controller\LoginController::perform_login_if_requested( )	.../LoginController.php:34

I'm still investigating.

tchalvak commented:
Added a commit to include global namespaces. I don't get how this code could have been working on your end if you had manually tested login/logout after changing it. Please manually retest areas of system that you actually changed, or create automatic tests for them, or both.

Beagle commented:
Rebased this pull request on top of the other merges so remove conflict. Squashed your additional namespaces commit into my previous commit on LoginController. Manually tested.
